lib/compat/wordpress-6.5/interactivity-api/interactivity-api.php

```diff
@@ -23,23 +23,40 @@
 	function wp_interactivity_process_directives_of_interactive_blocks( $parsed_block ) {
 		static $root_interactive_block = null;
 
+		/**
```

Comment:
sorry for the noise and the nitpick. I think this code will pass fine, but for future reference, these are PHPDoc comments and I think we only want multi-line comments, which start with `/*` and not with `/**`

Reply:
Ohhh, ok. I'm learning a lot about comments lately 🤷‍♂️😄

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/58066/commits/c3c87d330b630ab75d1b4625d983d73b48c1848a.